cmd/zpool/zpool_main.c

```diff
@@ -409,7 +409,7 @@ get_usage(zpool_help_t idx)
 		    "[<device> ...]\n"));
 	case HELP_STATUS:
 		return (gettext("\tstatus [-c [script1,script2,...]] "
-		    "[-igLpPstvxD]  [-T d|u] [pool] ... \n"
+		    "[-idgLpPstvxD]  [-T d|u] [pool] ... \n"
```

Comment:
Didn't find man pages update for -d, sorry if I missed something

Reply:
Good catch and thank you. I went ahead and updated the man page. Let me know if you think it should be updated further.